lib/icalendar/util/deserialize.ex

```diff
@@ -85,6 +85,40 @@ defmodule ICalendar.Util.Deserialize do
   ) do
     %{acc | location: desanitized(location)}
   end
+  def parse_attr(
+    %Property{key: "COMMENT", value: comment},
+    acc
+  ) do
+    %{acc | comment: desanitized(comment)}
+  end
+  def parse_attr(
+    %Property{key: "STATUS", value: status},
+    acc
+  ) do
+    %{acc |
+      status: status |> desanitized() |> String.downcase() |> String.to_atom()
```

Comment:
Can we remove the `String.to_atom/1` please? Turning random input to atoms is an easy way to leak atoms which can crash the VM.